### PR TITLE
Updating AzureRm.Profile version dependency to 5.6.0

### PR DESCRIPTION
**Pull request by bganapa**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by bganapa** [comment]
ps-sign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1260

File: src/StackAdmin/AzureStack/AzureStack.psd1 [hunk 1]
```diff
@@ -122,8 +122,23 @@
 
             # ReleaseNotes of this module
             ReleaseNotes = '## 2018.09 - Version 1.5.0
```
Comment (deathly809) [inline src/StackAdmin/AzureStack/AzureStack.psd1]: I thought we were going to remove the Version.

**Comment by bganapa** [comment]
@markcowl @cormacpayne  This should be good to merge , successful ps-sign is here https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1270/

File: src/StackAdmin/Azs.AzureBridge.Admin/Module/Azs.AzureBridge.Admin/Azs.Azurebridge.Admin.psd1 [hunk 2]
```diff
@@ -56,8 +56,8 @@ Licensed under the MIT License. See License.txt in the project root for license
     # ProcessorArchitecture = ''
 
     # Modules that must be imported into the global environment prior to importing this module
-    RequiredModules   = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.5.1'; },
-        @{ModuleName = 'AzureRM.Resources'; ModuleVersion = '6.0.2'; })
+    RequiredModules   = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.6.0'; },
+        @{ModuleName = 'AzureRM.Resources'; RequiredVersion = '6.0.2'; })
```
Comment (cormacpayne) [inline src/StackAdmin/Azs.AzureBridge.Admin/Module/Azs.AzureBridge.Admin/Azs.Azurebridge.Admin.psd1]: @bganapa what's the reason to depend on a specific version of `AzureRM.Resources` for these modules?

Comment (bganapa) [reply src/StackAdmin/Azs.AzureBridge.Admin/Module/Azs.AzureBridge.Admin/Azs.Azurebridge.Admin.psd1]: AZURESTACK supports only a particular version of AzureRm.Resources, we have a copy in the stackadmin folder. We cannot take dependency on the latest as the api-version will be that of latest azure.

**Pull request merged**